client/src/components/ExtraUsers/ExtraUsers.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import "./ExtraUsers.css";
+
+
+// Class Based React Component
+function ExtraUsers(props) {
+  let extraUsersValue = null
+
+  function howManyMore(theArray) {
+    if (theArray.length > 4) {
+      let extraValue = theArray.length - 4
+      return extraValue
+    }
+  }
```

Comment:
This doesn't need to be in a function. Shouldn't this only get used by the parent component if there are extras? In that case, this should just take the number of extra users and render that number - keep it simple!